superset/viz.py

```diff
@@ -939,7 +939,7 @@ def to_series(self, df, classed='', title_suffix=''):
             if isinstance(series_title, string_types):
                 series_title += title_suffix
             elif title_suffix and isinstance(series_title, (list, tuple)):
-                series_title = series_title + (title_suffix,)
+                series_title = str(series_title[-1]) + title_suffix
```

Comment:
Can the series title contain unicode characters? If so should this be `six.string_types(...)` rather than `str(...)`

Reply:
good call